README.md

```diff
@@ -247,3 +247,25 @@ const container = await new GenericContainer("redis")
   .withWaitStrategy(Wait.forHealthCheck())
   .start();
 ```
+
+## Log Config
+
+You can override the logging driver used by docker. This might be necessary
+when the default driver of your docker host does not support reading logs
+and you want to use the `Wait.forLogMessage` wait strategy.
+
+This is the same as [--log-driver on docker run](https://docs.docker.com/config/containers/logging/configure/#configure-the-logging-driver-for-a-container).
+
+```javascript
+const { GenericContainer, Wait } = require("testcontainers");
+
+const container = await new GenericContainer("redis")
+  .withExposedPorts(6379)
```

Comment:
The `Wait` import and `withExposedPorts` could be removed to simplify the example